client/src/pages/products/ProductPage.jsx

```diff
@@ -1,7 +1,44 @@
-import AddProduct from './AddProduct';
+import { useState, useEffect } from 'react';
+import { Button } from 'react-bootstrap';
+import ListOfItems from '../../components/ListOfItems';
 
 const ProductPage = () => {
-  return <AddProduct/>
+  const [products, setProducts] = useState([]);
+
+  // Funkcja do pobierania produktów
+  const fetchItems = async () => {
+    try {
+      const response = await fetch(
+        'http://localhost:5000/products/get-all-products',
```

Comment:
I suggest to rename endpoints, because it may be better to being compliant with the standard.

Get all products:
`/products` - GET method

Get product by ID:
`/products/:id` - GET method

Create product:
`/products` - POST method

Other endpoints could be named similar.

Reply:
Yup, will do that.